java/sage/Show.java

```diff
@@ -1536,9 +1502,9 @@ public static String getImageIdUrl(int packedImageId, int rootId, int altFilmId,
   long duration;
   Stringer title;
   volatile String episodeNameStr;
-  byte[] episodeNameBytes;
+  byte[] episodeNameBytes; /* deprecated no longer used */
```

Comment:
Remove both of these so we know for sure they are no longer used...because if they are it won't compile. :)